src/client/Microsoft.Identity.Client/Platforms/Android/BrokerRequest.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.Identity.Client.Platforms.Android
+{
+    [DataContract]
```

Comment:
Please note that I've merged changes to move away from DataContract serialization. Rebase to master and replace `DataContract` with `JsonObject` aND `DataMember` with `JsonProperty`.